### ISSUE TITLE
Inappropriate method call child indentation reported

### ISSUE THREAD

**Issue by dgervalle** [checkstyle/checkstyle#132]
Checkstyle 5.7 never gets happy about the indentation of the following code, reporting that the line `method().foo = "bar";` is never properly indented with a message `method call child at indentation level 8 not at correct indentation, 12`. If you move it to 12, it says `method call child at indentation level 12 not at correct indentation, 8`.
(I have checked from IDEA IntelliJ's CheckStyle IDEA plugin, as well as from EclipseCS)

It looks similar to issue #129, since it is not precisely the same, I had prefer a separate issue. Hope this helps.

Here is a minimal valid java:

``` java
public class Test {
    private static final class C {
        private String foo;
    }

    C method() {
        return new C();
    }

    void call() {
        method().foo = "bar";
    }
}
```

But this is enough to reproduce (not valid java, however)

``` java
public class Test {
    void call() {
        method().foo = "bar";
    }
}
```

The config file is

``` xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
    "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
    "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
<module name="Checker">
  <module name="TreeWalker">
    <module name="Indentation"/>
  </module>
</module>
```
